contracts/DEX.sol

```diff
@@ -24,13 +26,19 @@ contract DEX {
         uint256 ssvAmount = amount / rate;
         cdtToken.transferFrom(msg.sender, address(this), amount);
         ssvToken.transfer(msg.sender, ssvAmount);
+        cdtAllowance[msg.sender] += amount;
```

Comment:
same here.
you can use a variable for it.

Reply:
fixed